src/mmw/js/src/data_catalog/views.js

```diff
@@ -469,23 +464,156 @@ var ResultDetailsView = Marionette.ItemView.extend({
             placement: 'right',
             trigger: 'click',
         });
-        this.$('[data-toggle="table"]').bootstrapTable();
     },
 
+    closeDetails: function() {
+        this.model.collection.closeDetail();
+    }
+});
+
+var ResultDetailsCinergiView = ResultDetailsBaseView.extend({
+    template: resultDetailsCinergiTmpl,
+});
+
+var ResultDetailsHydroshareView = ResultDetailsBaseView.extend({
+    template: resultDetailsHydroshareTmpl,
+});
+
+var ResultDetailsCuahsiView = ResultDetailsBaseView.extend({
+    template: resultDetailsCuahsiTmpl,
+
     templateHelpers: function() {
         var id = this.model.get('id'),
-            location = id.substring(id.indexOf(':') + 1);
+            location = id.substring(id.indexOf(':') + 1),
+            fetching = this.model.get('fetching'),
+            error = this.model.get('error'),
+            last_date = this.model.get('end_date');
+
+        if (!fetching && !error) {
+            var variables = this.model.get('variables'),
+                last_dates = variables.map(function(v) {
+                        var values = v.get('values');
+
+                        if (values.length > 0) {
+                            return new Date(values.last().get('datetime'));
+                        } else {
+                            return new Date('01/01/1900');
```

Comment:
What's the significance of this date? Sometimes I see "last updated 111 years ago", which leads me to believe there might be some that are older than 1900.

Reply:
This "default" value was chosen to match the `DATE_MIN` set [here](https://github.com/WikiWatershed/model-my-watershed/blob/fe10d235207e902e7be5cebd8acc3a542e2a0a02/src/mmw/apps/bigcz/clients/cuahsi/search.py#L30), but is otherwise arbitrary. Maybe we should think of a better way to say "Never updated".